src/components/NAlert/__snapshots__/NAlert.spec.js.snap

```diff
@@ -0,0 +1,43 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
```

Comment:
Didn't need to be in the repo. Merging for now, will remove from develop base branch